forte/ontology_specs/race_qa.json

```diff
@@ -4,12 +4,12 @@
     "base_ontology.json"
   ],
   "additional_prefixes": [
-    "ftx.onto.race_qa"
+    "ftx.mcqs"
```

Comment:
I feel like we can keep these to be `ftx.onto` if they don't conflict with our projects? Making fewer changes might be easier for us.

Reply:
Makes sense, reverted these changes. Now, building all example ontologies in `ftx.onto.*`